governance.md

```diff
@@ -106,15 +107,28 @@ This is the purpose of Working Groups (WG). The intent is to make
 Working Groups relatively easy to create and to deprecate, once
 inactive.
 
-To propose a new working group, first find a SIG to sponsor the group. 
+Working groups do not own any code or subprojects. Instead, they are a place for
```

Comment:
we kept getting hung up on this during the WG discussion. I think everyone agreed working groups can create and prototype. Just that the code landing in Kubernetes is contingent on SIG/subproject approval. "own any code" is confusing in this meaning.

Reply:
Agreed.  I think that this needs to be addressed further up in the doc too.

Reply:
How about:

“Since WG are intended to have a definite lifetime, to prevent orphaned code at the dissolution of a WG all code in Kubernetes must have a SIG to own and maintain it.  The sponsoring SIGs for a WG are expected to own and manage the code created for Kubernetes by a WG, and have authority on whether code created in a WG should become part of Kubernetes”

Or similar?

Reply:
Perhaps a bit wordy, but otherwise fine.  It can be word-smithed in a followup PR, IMO.

Reply:
@philips @smarterclayton @quinton-hoole 

We have a repository policy:
https://github.com/kubernetes/community/blob/master/github-management/kubernetes-repositories.md

This PR should not change it.